style.css

```diff
@@ -72,3 +68,193 @@ p {
   background-repeat: no-repeat;
   background-position: right top;
 }
+
+.section-1 {
+  margin-top: 40px;
+  padding: 2em;
+  background: #f7f7f9;
+  border: 2px solid #e1e3e8;
+  height: 100%;
+  width: 100%;
+}
+
+.first-card {
+  margin-top: 80px;
+  background: #fff;
+  padding: 1rem;
+  border: 2px solid #e1e3e8;
+  box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1);
+  border-radius: 1rem;
+  height: auto;
+  width: 100%;
+}
+
+h1 {
+  font-size: 30px;
+  color: #172b4d;
+}
+
+.details {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  width: 300px;
+  color: #abd4c0;
+}
+
+.card-img {
+  width: 100%;
+}
+
+h3 {
+  color: #7d899c;
+  font-size: 15px;
+}
+
+h4 {
+  color: #848fa2;
+  font-size: 13px;
+}
+
+.languages {
+  display: flex;
+  gap: 20px;
```

Comment:
- Kindly reduce the gap, maybe to 6px or 7px, to fit the languages inside card on the mobile version